pydantic_ai_slim/pydantic_ai/_cli.py

```diff
@@ -154,6 +172,21 @@ def cli(  # noqa: C901
     argcomplete.autocomplete(parser)
     args = parser.parse_args(args_list)
 
+    # Handle web subcommand
+    if args.command == 'web':
+        try:
+            from clai.chat.cli import run_chat_command
+        except ImportError:
+            print('Error: clai web command is only available when clai is installed.')
```

Comment:
probably can be improved by adding how to install, but not sure whether we should suggest `uv add clai` or rather tell the user to `uv tool install clai`